lib/camera_api/admin/power/show_off_delay_button.dart

```diff
@@ -25,13 +26,20 @@ class ShowOffDelayButton extends StatelessWidget {
           }
         });
         var formattedResponse;
+        var formattedReqRes;
+
         try {
           var fullResponse = await http.post(url,
               headers: {"Content-Type": "application/json; charset=utf-8"},
               body: body);
-          var offDelayMessage = '\n65535 will disable power off';
+          var offDelayMessage = '\n65535 will disable power off, bitches!';
```

Comment:
might be better to change the message as the repository is public.

Reply:
Did a PR with fixed message